.github/workflows/workflow.yaml

```diff
@@ -34,3 +34,29 @@ jobs:
       - run: yarn typecheck
       - run: yarn lint
       - run: yarn format:check
+  publish:
+    name: Publish to NPM
+    if: github.ref == 'refs/heads/main'
```

Comment:
Does semantic-release ensure it doesn't run for a commit by semantic release or do we need a condition here?

Reply:
it does

Reply:
puts [no-release] in the commit body I think